tests/src/Functional/ContentProjectRenderTest.php

```diff
@@ -291,4 +320,102 @@ protected function getStorage(string $entity_type_id): EntityStorageInterface {
     return \Drupal::entityTypeManager()->getStorage($entity_type_id);
   }
 
+  /**
+   * Updates the project date, saves the node, and refreshes the page.
+   *
+   * @param \Drupal\node\NodeInterface $node
+   *   Node to update.
+   * @param string $begin
+   *   Start date string.
+   * @param string $end
+   *   End date string.
+   */
+  protected function setProjectDateRange(NodeInterface $node, string $begin, string $end): void {
+    $node->oe_project_dates = [
+      'value' => (new DrupalDateTime($begin, 'Europe/Brussels'))->format('Y-m-d'),
+      'end_value' => (new DrupalDateTime($end, 'Europe/Brussels'))->format('Y-m-d'),
+    ];
+    $node->save();
+    $this->drupalGet($node->toUrl());
+  }
+
+  /**
+   * Asserts that the d-none class has been removed from project status.
+   */
+  protected function assertProjectStatusRevealed(): void {
+    $status_section = $this->assertSession()->elementExists('css', '.bcl-project-status');
+    $this->assertFalse($status_section->hasClass('d-none'));
+  }
+
+  /**
+   * Asserts start and end timestamp in project status section.
+   *
+   * @param string $begin
+   *   Expected start date string as 'Y-m-d H:i:s'.
+   * @param string $end
+   *   Expected end date string as 'Y-m-d H:i:s'.
+   */
+  protected function assertProjectStatusTimestampsAsDateStrings(string $begin, string $end): void {
```

Comment:
Btw here I use a hard-coded timezone, but in `assertTimestampAsDateString()` I have a timezone param.
I'd say the rationale is that `assertTimestampAsDateString()` is designed as a more generic function, even if used only once.

Reply:
I dont mind